### ISSUE TITLE
Ubuntu16.04 or ubuntu18.04 is ok?

### ISSUE THREAD

**Issue by zhy102** [kdkalvik/Atheros-CSI-Tool#1]
i want to konw whether ubuntu16.04 or ubuntu18.04 with 4.15 kernel support Atheros csi tool?

**Comment by itskalvik** [comment]
Pretty much any distribution will work as long as you have the right kernel. So yes, either should work but, I believe I only tested 16.04.

**Comment by zhy102** [comment]
> Pretty much any distribution will work as long as you have the right kernel. So yes, either should work but, I believe I only tested 16.04.

Thanks. The right kernel should mean kernel 4.15.0(not kernel 4.1.10+)? And how i install this kernel ?  i will try it by ubuntu16.04.

**Comment by itskalvik** [comment]
Yes, by right kernel, I meant the kernel provided in my repo. If you want to run the driver on an even newer kernel, you can follow my commit history and make the same changes to the latest kernel. The installation instructions are in the README file.
